scm-ui/ui-webapp/public/locales/de/commons.json

```
@@ -124,5 +124,11 @@
   },
   "importLog": {
     "title": "Importprotokoll"
+  },
+  "notifications": {
+    "toastTitle": "Benachrichtigung",
```

```suggestion
    "toastTitle": "System Nachricht",
```

Not sure about this one. What do you think?

---

Why system? What if we build a plugin, which allows a user sending a notification to another user?